src/select/styled-components.js

```diff
@@ -35,6 +35,16 @@ function getControlPadding(size = SIZE.default, sizing, type) {
   }[size];
 }
 
+function ellipsisText() {
```

Comment:
I would like to reuse in other components. We can put it here src/styles/util.js

Reply:
It we are going to reuse we may want to drop the `maxWidth` value